include/ebpf_api.h

```diff
@@ -491,6 +491,26 @@ extern "C"
         _Out_ ebpf_program_type_t* program_type,
         _Out_ ebpf_attach_type_t* expected_attach_type);
 
+    /**
+     * @brief Get the name of a given program type.
+     *
+     * @param[in] program_type Program type.
+     *
+     * @returns Name of the program type, or NULL if not found.
+     */
+    _Ret_maybenull_z_ const char*
```

Comment:
Does the caller need to free the string? Recommend document the contract.

Reply:
the caller does NOT need to free, just like `bpf_object__name()` etc